mpdev/internal/apply/deployment_manager.go

```diff
@@ -23,13 +23,14 @@ import (
 
 	"github.com/GoogleCloudPlatform/marketplace-tools/mpdev/internal/util"
 	"github.com/pkg/errors"
+	"gopkg.in/yaml.v3"
 )
 
 // DeploymentManagerAutogenTemplate generates a deployment manager template
 // given an autogen.yaml file.
 type DeploymentManagerAutogenTemplate struct {
 	BaseResource
-	AutogenFile string
+	AutogenSpec interface{}
```

Comment:
Can you update the PR description as well?

Reply:
Done